roles/aws/admin-route53/tasks/adm-record.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Ensure the record exist
+  route53:
+    aws_access_key: "{{ aws_access_key }}"
+    aws_secret_key: "{{ aws_secret_key }}"
+    zone: "{{ item.0.zone }}"
+    record: "{{ item.1.hostname }}.{{ item.0.zone }}"
```

Comment:
Per the test data, the record should be for example hostname + sub-domain, not hostname + zone. The former seems more aligned with for example nsupdate. Can you please double check how this is handled?

Reply:
For the route53 module, ansible is expecting 'The full DNS record to create or delete' be provided to 'record' option, so this should be fine.